third_party/move/move-model/src/ast.rs

```diff
@@ -963,13 +963,22 @@ impl ExpData {
     where
         F: FnMut(VisitorPosition, &ExpData) -> bool,
     {
-        let _ = self.visit_positions_impl(&mut |x, e| {
+        self.visit_positions_with_return_val(visitor);
+    }
+
+    /// Same as `visit_positions`, but returns false iff any visit of a subexpression returns false
+    pub fn visit_positions_with_return_val<F>(&self, visitor: &mut F) -> bool
```

Comment:
`visit_positions_visitor_always_returns_true` might be a little more meaningful.  (or `visit_positions_always_returns_true`), so that the meaning is clear at the places where it is used.

Reply:
done